bigframes/core/window_spec.py

```diff
@@ -152,3 +152,8 @@ class WindowSpec:
     ordering: Tuple[orderings.OrderingExpression, ...] = tuple()
     bounds: Union[RowsWindowBounds, RangeWindowBounds, None] = None
     min_periods: int = 0
+
+    @property
+    def row_bounded(self):
+        # relevant for determining if window requires total ordering for determinism.
```

Comment:
Nit: This could be transformed into a docstring. I like that you shared the "why" but I'd also like a little more explanation on what row_bounded means.

Reply:
changed to docstring